src/components/WhimsicalInstaller/WhimsicalInstaller.js

```diff
@@ -278,17 +279,29 @@ class WhimsicalInstaller extends PureComponent<Props, State> {
     );
 
     // When files get near the folder, the folder "mouth" opens up.
-    // As they get even closer, the mouth closes again.
     if (
       !isFolderOpen &&
       this.areFilesWithinRangeOfFolder(freeFlyingFileIds, FOLDER_OPEN_RADIUS)
     ) {
       this.setState({ isFolderOpen: true });
-    } else if (
-      isFolderOpen &&
-      this.areFilesWithinRangeOfFolder(fileIdsBeingInhaled, FOLDER_CLOSE_RADIUS)
-    ) {
-      this.setState({ isFolderOpen: false });
+    }
+
+    // If files are within "swallow range", the folder closes.
+    // Alternatively, if they get too far away (if they escape), it should
+    // also close.
+    if (isFolderOpen) {
+      const isWithinSwallowRange = this.areFilesWithinRangeOfFolder(
+        fileIdsBeingInhaled,
+        FOLDER_CLOSE_RADIUS
+      );
+
+      const haveFilesEscaped =
+        !isWithinSwallowRange &&
+        !this.areFilesWithinRangeOfFolder(activeFileIds, FOLDER_OPEN_RADIUS);
```

Comment:
Small tweak when I noticed that if a folder managed to escape (say if you drag it near the folder but not within its gravity), the folder would stay open awkwardly.